PSRabbitMq/Send-RabbitMqMessage.ps1

```diff
@@ -63,21 +102,51 @@
         [string]$Key,
 
         [Parameter(Mandatory=$true,ValueFromPipeline=$true)]
+        [Alias('Payload')]
         $InputObject,
 
         [switch]$Persistent,
 
         [Int32]$Depth = 2,
 
-        [PSCredential]$Credential,
+        [PSCredential]
+        [System.Management.Automation.Credential()]
+        $Credential,
 
         [System.Security.Authentication.SslProtocols]$Ssl,
 
-        [parameter(Mandatory = $false)]
-        [string]$vhost
+        [string]$vhost = '/',
+
+        [ValidateSet('application/clixml+xml','application/json','text/xml', 'text/plain')]
```

Comment:
If I understand you suggest removing the ValidateSet to allow other content type: Yep, that makes sense and is less restrictive.
In that case, the Default case from ConvertFrom-RabbitMQDelvivery should probably return the raw bytes, and we should probably support other encoding than UTF8 (I think there's a property for that).
For renaming the Parameter to SerializeAs, that makes sense, probably need to add an option for 'None' so that someone can plug their own and define the ContentType of their choice without changing the Module.